integration-tests/provisioning/conftest.py

```
@@ -0,0 +1,132 @@
+REPO_ROOT = Path(__file__).resolve().parents[2]
+HELPER_SOURCE = REPO_ROOT / "packaging" / "libexec" / "repovec-qdrant-api-key"
+
+
+@pytest.fixture()
+def patched_helper(tmp_path: Path) -> Path:
+    """Return a tmp-rooted copy of the helper script.
+
+    The helper hard-codes ``/etc/repovec`` paths. For cmd-mox tests we cannot
+    write there (and would not want to), so we rewrite the two path constants
+    (``CONFIG_DIR`` and ``KEY_FILE``) near the top of the script to point into
+    ``tmp_path``. The rest of the file – including the logic we are trying to
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)